staging/src/k8s.io/csi-translation-lib/plugins/in_tree_volume.go

```diff
@@ -165,6 +212,101 @@ func translateTopology(pv *v1.PersistentVolume, topologyKey string) error {
 	return nil
 }
 
+// isTopologyLabelGA checks if the topology label used in this PV is GA
+// and return the zone/region label used. If no labels are found, check
+// NodeAffinity to find the label
+func isTopologyLabelGA(pv *v1.PersistentVolume) (bool, string, string) {
```

Comment:
a better name for this function may be something like `getTopologyLabel`. Does it need to return a bool? It always matches with the string.

Reply:
The bool might become handy in some cases. It does not hurt to return it here I think.